templates/ehub.json

```diff
@@ -53,6 +53,34 @@
             "metadata": {
                 "description": "Please use the actual name if using existing event hub."
             }
+        },
+         "newEhubMaxThroughputUnits": {
```

Comment:
I think we need to have all parameter names formatted in the same way `Event Hub Max Throughput Units`etc.

Reply:
ok will make the change